ui/pages/confirmations/components/confirm/footer/footer.tsx

```diff
@@ -73,21 +168,27 @@ const Footer = () => {
       >
         {t('cancel')}
       </Button>
-      <Button
-        block
-        data-testid="confirm-footer-confirm-button"
-        onClick={onSubmit}
-        size={ButtonSize.Lg}
+      {alertModalVisible ? (
+        <MultipleAlertModal
+          alertKey={unconfirmedAlerts[0]?.key}
+          ownerId={alertOwnerId}
+          onFinalAcknowledgeClick={handleCloseModal}
+          onClose={handleCloseModal}
+        />
+      ) : null}
+      <ConfirmFooterButton
+        hasUnconfirmedAlerts={hasUnconfirmedAlerts}
+        unconfirmedAlerts={unconfirmedAlerts}
```

Comment:
We're only passing the unconfirmed alerts here, but don't we want all the alerts (even if confirmed) to influence the style of the button?

Reply:
True, passing all the alerts to get the style of the button.